main.py

```diff
@@ -1,7 +1,45 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for first, last in actors:
+        Actor.objects.create(first_name=first, last_name=last)
```

Comment:
get_or_create instead of create: Using get_or_create ensures that you don’t create duplicates if the records already exist. It also avoids raising an error if you attempt to create a record that already exists.

Reply:
okey, but i already saw, that it's the next topic